Sources/SUIComponents/Atoms/Button+HMRC.swift

```diff
@@ -28,8 +28,8 @@ extension Button {
         switch style {
             case .primary:
                 self.buttonStyle(PrimaryButtonStyle())
-            case .secondary(let padding):
-                self.buttonStyle(SecondaryButtonStyle(padding))
+            case let .secondary(padding, image):
```

Comment:
As discussed - lets lose this until we make the [IconButtonView Component](https://jira.tools.tax.service.gov.uk/browse/HMA-4860)